scss/style.scss

```diff
@@ -10,7 +10,9 @@
 @import
 	"atoms/icons/icons",
 	"atoms/typography/typography",
-	"atoms/buttons/buttons";
+	"atoms/buttons/buttons",
+	"atoms/input_styles/input_styles";
```

Comment:
everything in our scss is styles mostly. keep the directory name simply 'inputs'